data/remnant/remnant 2 side missions.txt

```diff
@@ -640,3 +640,57 @@ mission "Remnant: Shattered Light 6"
 			`	Taely gestures take a tone of curiosity. "One thing that you could help us with, if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`
 			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at the Tycho Crater shipyard on Luna... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
 			`	"Someday, Captain <first>, much may come from this. But for now, it will be months before we have analyzed everything. Thank you for helping recover a piece of our history, Captain."`
+
+
+
+mission "Remnant: Scent of Rain 1"
+	landing
+	name "Search for the unknown Korath ship."
+	description "Chilia has asked you to find the Korath ship that can carry three 'olofez."
+	source
+		government "Remnant"
+	destination "Caelian"
+	to offer
+		has "Remnant: Cognizance 25: done"
+	npc "scan outfits"
+		government "Korath"
+		personality waiting target
+		system "Kor Zena'i"
+		ship "Tubfalet" "Ra'puk At 295"
+		dialog `In poor lighting, this ship might have passed for a Palavret, but major structural differences suggest this is at least a different model, or perhaps an entirely different ship. Time to report this to Chilia.`
```

Comment:
This dialog comes up literally just after you scan it. Are these structural differences picked out by eye or the outfit scanner? It may be nice for once to see the player seeing something else other than lists of stuff in their scans (even if it is just in universe and not mechanically done)

Reply:
I made a slight tweak to this dialog to make it clear that it is scanning that reveals the structural differences.

Reply:
The model does have a fair bit of visible varying structures that the base Raider does not have. It's not so much internal as it is simply the base of the ship itself being different visually (one extra asymmetrical bay, an asymmetrical gun pod, additional meat by the firelight ports, additional color rings, additional crystal-like structures, etc..

Reply:
Exactly. This dialog is making the point that all that visual stuff isn't just surface stuff - it is symptomatic of deep internal changes.